index.html

```diff
@@ -1,251 +1,285 @@
+
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+
+<body>
+    <div>
+        <nav class="nav-links">
+            <ul class="nav-mobile">
+                <li><a href="/">Tolu's Logo</a></li>
+                <li class="handburger" ><a href="/"><i class="fas fa-bars hamburger-icon menu"></i></a></li>
+            </ul>
+            <ul class="nav-desktop">
+                <li ><a href="/">Portfolio</a></li>
+                <li><a href="/">About</a></li>
+                <li><a href="/">Contact</a></li>
+                <li><a href="/"><i class="fas fa-envelope"></i></a></li>
+            </ul>
+        </nav>
+    </div>
+
+    <main>
+        <section>
+            <div class="main-section">
+                <div class="headline">
+                    <h1>Hey There.<br> I am Tolu</h1>
+                    <h4>I am a Software Developer</h4>
+                    <p>I can help you build a product, feature or website. Look through some of my work and experience! If
+                        you like what you see and have a project you need coded, do not hesitate to contact me.</p>
+                </div>
+    
+                <div class="social-icons">
+                    <ul>
+                        <li><a href="/"><i class="fab fa-github"></i></a></li>
+                        <li><a href="/"><i class="fab fa-linkedin-in"></i></a></li>
+                        <li><a href="/"><i class="fab fa-angellist"></i></a></li>
+                        <li><a href="/"><i class="fab fa-twitter"></i></a></li>
+                        <li><a href="/"><i class="fab fa-medium-m"></i></a></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="desktop-main-section">
+                <div>
+                    <ul class="desktop-social-icons">
+                        <li><a href="/"><i class="fab fa-github"></i></a></li>
+                        <li><a href="/"><i class="fab fa-linkedin-in"></i></a></li>
+                        <li><a href="/"><i class="fab fa-angellist"></i></a></li>
+                        <li><a href="/"><i class="fab fa-twitter"></i></a></li>
+                        <li><a href="/"><i class="fab fa-medium-m"></i></a></li>
+                    </ul>
+                </div>
+                <div class="desktop-header">
+                    <h1>Hey There.<br> I am Tolu</h1>
+                    <h4>I am a Software Developer</h4>
+                    <p>I can help you build a product, feature or website. Look<br>through some of my work and experience! If
+                        you like what<br>you see and have a project you need coded,<br>do not hesitate to contact me.</p>
+                </div>
+            </div>  
+        </section>
+
+        <section class="main-work-container">
+            <div class="work-section">
+                <h2 class="work-h1">My Recent Works</h2>

```

Comment:
- [ ] This heading should be placed on one line for desktop screens.
![Screenshot from 2021-08-20 15-14-21](https://user-images.githubusercontent.com/65501535/130231501-173cd68b-f13d-42e3-a01a-04aaceeddeaa.png)

Original
![Screenshot from 2021-08-20 15-15-11](https://user-images.githubusercontent.com/65501535/130231575-1aa9234f-e6a2-4811-a6cf-b9856dac642a.png)